src/Workspaces/VisualBasic/Portable/CodeStyle/VisualBasicCodeStyleOptions.vb

```diff
@@ -23,5 +23,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeStyle
             New CodeStyleOption(Of String)(String.Join(",", PreferredModifierOrderDefault.Select(AddressOf SyntaxFacts.GetText)), NotificationOption.None),
             EditorConfigStorageLocation.ForStringCodeStyleOption("visual_basic_preferred_modifier_order"),
             New RoamingProfileStorageLocation($"TextEditor.%LANGUAGE%.Specific.{NameOf(PreferredModifierOrder)}"))
+
+        Public ReadOnly PreferInferredTupleNames As [Option](Of CodeStyleOption(Of Boolean)) = New [Option](Of CodeStyleOption(Of Boolean))(
```

Comment:
see how the other options in this file are used.  You'll need to update some more code if you add this (same applies for C#).